scripts/run-operator.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# (c) Copyright [2021-2022] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+REPO_DIR=$(dirname $SCRIPT_DIR)
+INT_TEST_OUTPUT_DIR=${REPO_DIR}/int-tests-output
+NS=$(kubectl config view --minify --output 'jsonpath={..namespace}' 2> /dev/null)
+CYAN='\033[0;36m'
+RED='\033[1;31m'
+NC='\033[0m'  # No color
+
+set -o errexit
+set -o pipefail
+
+# This is a script that will run the operator from your local environment. It
+# won't run it in a k8s pod. This has the advantage of starting quicker, since
+# we can use the Go cache to speed up the build times. The downside is that the
+# webhook has to be disabled. There is no way for k8s to send webhook requests
+# to your local environment.
+
+function usage {
+    echo "$0 [-l <log_dir>] [-v]"
+    echo
+    echo "Options:"
+    echo "  -l <log_dir>        Log directory.   default: $INT_TEST_OUTPUT_DIR"
+    echo "  -v                  Verbose output"
+    exit 1
+}
+
+OPTIND=1
+while getopts l:hv opt; do
+    case ${opt} in
+        l)
+            INT_TEST_OUTPUT_DIR=${OPTARG}
+            ;;
+        h)
+            usage
+            ;;
+        v)
+            set -o xtrace
+            ;;
+        \?)
+            echo "ERROR: unrecognized option: -$opt"
+            usage
+            ;;
+    esac
+done
+
+function logGeneric {
+    printf $1
+    shift
+    printf $(date "+%D")
+    printf " "
+    printf $(date "+%T")
+    printf " [$1]  "
+    shift
+    for i in $*; do
+        printf "$i "
+    done
+    printf "$NC\n"
+
+}
+
+function logInfo {
+    logGeneric ${CYAN} "ℹ" $@
+}
+
+function logError {
+    logGeneric ${RED} "!" $@
+}
+
+cd $REPO_DIR
+
+mkdir -p $INT_TEST_OUTPUT_DIR
+OP="${INT_TEST_OUTPUT_DIR}/local-verticadb-operator.log"
+
+if helm list -o json | jq --exit-status '.[] | select(.chart|test("^verticadb-operator")) | .name'
```

Comment:
Why do we check this if we run the operator in our local env?

Reply:
This is just to make sure the operator isn't already deployed through a helm chart. I wasted a bunch of time because I was running the operator locally and had it deployed in helm too. This is just a check to make sure two operators aren't running at the same time.